packages/module/patternfly-docs/content/extensions/chatbot/design-guidelines.md

```diff
@@ -222,22 +296,36 @@ There are a few display modes that users can choose when interacting with a Chat
 ![ChatBot in overlay mode.](./img/overlay.svg)
 </div>
 
-2. **Docked:** Anchors the ChatBot to the side of the page content. When docked, the ChatBot window is persistent, and cannot be toggled.
+2. **Docked:** Anchors the ChatBot to the side of the page, on top of the page content. When docked, the ChatBot window is persistent, and cannot be toggled. To minimize the ChatBot, users will need to return to overlay mode. 
 
 <div class="ws-docs-content-img">
 ![ChatBot in docked mode.](./img/docked.svg)
 </div>
 
-3. **Full screen:** Fills the screen with the ChatBot window.
+3. **Embedded:** Places the ChatBot within a product as its own page. An embedded ChatBot could be displayed in the product's navigation menu.
+
+<div class="ws-docs-content-img">
+![ChatBot embedded in a product.](./img/embedded.svg)
+</div>
+
+4. **Full screen:** Fills the screen with the ChatBot window.
 
 <div class="ws-docs-content-img">
 ![ChatBot in full-screen mode.](./img/fullscreen.svg)
 </div>
 
-4. **Embedded:** Places the ChatBot within a product as its own page. An embedded ChatBot could be displayed in the product's navigation menu.
+5. **Inline drawer**: Places the ChatBot to the side of the page, inline with the page content.
```

Comment:
So this is something I want to be very clear about. (I talked to Nicole about this.) What are actually providing is a chatbot that fits nicely and performs nicely in a drawer. We do not provide the drawer. The drawer is up to the product. It's complicated for us to provide a drawer that will work for everyone in the world.

Reply:
This is what I changed to: _"A ChatBot that can fit and perform within a drawer. This drawer can look different for each product, but will often be placed to the side of the page, inline with the page content."_ 

I also added a similar snippet beside the inline drawer demo, lmk if you have any concerns with that!

Reply:
That's perfect I think. Thank you!